source/errors.rst

```diff
@@ -170,7 +170,7 @@ server is running.
 
   {
     "status": 401,
-    "error": "Authentication failure",
+    "error": "Authentication Failure",
```

Comment:
https://github.com/osuosl/timesync-node/blob/develop/src/errors.js#L154 shows `'Authentication failure'` as the correct form.